src/benchmarks/micro/libraries/Microsoft.Extensions.Configuration/GetChildKeysTests.cs

```diff
@@ -0,0 +1,111 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Memory;
+using BenchmarkDotNet.Attributes;
+
+namespace Benchmark
+{
+    [BenchmarkCategory(Categories.Libraries)]
+    public class GetChildKeysTests
+    {
+        private ChainedConfigurationProvider _chainedConfig;
+        private ChainedConfigurationProvider _chainedConfigEmpty; 
+        private ChainedConfigurationProvider _chainedConfigWithSplitting;
+        private ChainedConfigurationProvider _chainedConfigWithCommonPaths;
+
+        [GlobalSetup(Targets = new[] {
+            nameof(AddChainedConfigurationNoDelimiter),
+            nameof(AddChainedConfigurationEmpty),
+            nameof(AddChainedConfigurationWithSplitting),
+            nameof(AddChainedConfigurationWithCommonPaths)
+        })]
```

Comment:
`GlobalSetup` without `Targets` provided is applied to all benchmarks within given type. Since this is what you want, there is no need to specify targets here.
```suggestion
        [GlobalSetup]
```